apps/webapp/src/app/invite/use-meeting-by-invite.ts

```diff
@@ -26,5 +26,6 @@ export function useMeetingByInvite(inviteId?: string): [Meeting|undefined, strin
     },
     [db, inviteId]
   )
+  document.title = meeting?.title ?? 'Cebulendarz';
```

Comment:
to znaczy, że ustawisz tytuł zawsze jako skutek uboczny pobierania spotkania. np na liście spotksń zawsze wrzucisz ostatnie które się pobrało, czyż nie? ;)